.gitchangelog.rc

```diff
@@ -186,9 +189,12 @@ def load_version():
     current_version = Version(version_code)
 
     if meta.major_version:
-        new_version = Version(f'{current_version.major + 1}.0')
+        new_version = Version(f'{current_version.major + 1}.0.0')
```

Comment:
We can probably omit the micro version from the major/minor versioning since it still has a `current_version.micro == 0`

Reply:
So, yes, but that only applies for python when it parses the version using the packaging module. Personally I like the look of this better. Plus, since it's treated as a string in most places, I feel like it would be better to be consistent in this way, no?

Reply:
Guess that's fair. Personally I prefer keeping the smallest valid string for it (only showing minor/major when no patch is needed) but that really is personal preference. lgtm then